r/R/dataset-partition.R

```diff
@@ -25,12 +25,17 @@
 #' `DirectoryPartitioning` describes how to interpret raw path segments, in
 #' order. For example, `schema(year = int16(), month = int8())` would define
 #' partitions for file paths like "2019/01/file.parquet",
-#' "2019/02/file.parquet", etc.
+#' "2019/02/file.parquet", etc.  In this scheme null values will be skipped.
+#' In the previous example, if the month was null, the files would be placed
+#' in 2019/file.parquet.  An error will be raised if an outer directory is
```

Comment:
`An error will be raised` is what I would say in Python.  Does R translate invalid Status into some kind of "raised error" or would it be more accurate to say "returned"?  Is the R terminology "thrown"?

Reply:
Error will be raised sounds fine to me.